app/javascript/src/emoji_selector.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener('click', (event) => {
```

Comment:
Just curious about the decision of using a vanilla JS file vs a Stimulus controller

Reply:
Agree with @jp524. Stimulus is a better approach and will keep with the pattern we are already using. :)